extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -58,9 +55,10 @@ private void processMethods(HttpServerResponse response, String allowMethodsValu
             }
 
             List<String> validRequestedMethods = new ArrayList<>();
-            for (HttpMethod configMethod : corsConfig.methods.get()) {
-                if (requestedMethods.contains(configMethod.name().toLowerCase())) {
-                    validRequestedMethods.add(configMethod.name());
+            List<String> methods = corsConfig.methods.get().stream().map(String::trim).collect(Collectors.toList());
```

Comment:
Maybe you can still convert them to `HttpMethod` Enum to avoid having invalid inputs?